cuBERT_topic_modelling/environment.yml

```diff
@@ -0,0 +1,419 @@
+name: rapids-21.10
```

Comment:
You dont need to spicy all dependencies.  

Follow something like this : https://github.com/rapidsai/rapids-examples/blob/main/shareable-dataframes/conda/shareable_dataframes.yml

Reply:
Also move it to conda/environment.yml